cookbook/specs/fvm.json

```diff
@@ -1831,21 +2833,165 @@
                             },
                             {
                               "name": "pruning",
+                              "parse_directive": {
+                                "function_tag": "GET_BLOCK_BY_NUM"
+                                },
                               "values": [
                                 {
-                                  "expected_value": ""
+                                  "latest_distance": 2800
```

Comment:
this should be the number of blocks in 24H.
Blocks24H = 24 * 60 * 60 / 30 (block time) = 2880

Reply:
done